libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -203,13 +205,14 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _sem->give();
     }
 
+    // log raw data if flag is set
```

Comment:
I'm not advocating if checking the flag should be done here or inside *get_dataflash* method - either would be fine to me. What I'm saying is that, since it is in another method, we shouldn't put comments here of what happens there - let's say we remove the check for the flag or that other constraints are added, this comment would then be wrong.